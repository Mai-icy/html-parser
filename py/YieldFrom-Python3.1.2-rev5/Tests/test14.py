#
#   Test conversion of send(None) to next()
#

def g():
	yield from range(3)

gi = g()
for x in range(3):
	y = gi.send(None)
	print("Yielded:", y)
