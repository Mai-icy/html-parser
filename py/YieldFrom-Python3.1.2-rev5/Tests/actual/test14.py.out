Yielded: 0
Yielded: 1
Yielded: 2
